dev-images.json

```diff
@@ -5,5 +5,6 @@
   "azure-cluster-api-controllers": "quay.io/ademicev/cluster-api-provider-azure:latest",
   "gcp-cluster-api-controllers": "quay.io/ademicev/cluster-api-provider-gcp:latest",
   "ibmcloud-cluster-api-controllers": "quay.io/kabhat/ibmcloud-cluster-api-controllers:dev",
+  "vsphere-cluster-api-controllers": "registry.build01.ci.openshift.org/ci-ln-kq6nx7b/release:latest",
```

Comment:
This won't work in the future as it is a temporary CI image that would've already been deleted by now.
It'd be best if you could pull this down and push it to your quay registry as a public img similarly to the others.

Reply:
My bad, forgot to change this back.